reference/debugging.md

```diff
@@ -0,0 +1,299 @@
+# Debugging
+
+If you’re experiencing issues with the Ambassador Edge Stack and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+First, create an example configuration for debugging demonstrations that are available throughout this document.
+
+Then, complete the following debugging options:
+ 
+* Check Ambassador Status
+* Review Ambassador Logs
+* Examine Pod and Container Contents
+* Customize the Envoy Configuration
+
+## Example Configuration
+
+To see a demonstration of how you can debug your Ambassador instance, follow the instructions to create an example mapping configuration.
+
+Note: The following assumes that you deployed Ambassador and the following services from the [quick start installation guide](../../user-guide/install) to a Kubernetes cluster.
+
+**In the command line:**
+
+1. Create a cluster in GKE with RBAC support enabled, and your user account configured correctly. Then run the following:
+
+    ```console
+    $ gcloud container clusters create ambassador-demo --preemptible
+    $ kubectl create clusterrolebinding cluster-admin-binding-new \
+    --clusterrole cluster-admin --user <your_user_name>
+    ```
+
+2. Deploy the latest version of Ambassador:
+
+    ```shell
+    $ kubectl apply -f https://www.getambassador.io/early-access/yaml/aes-crds.yaml
```

Comment:
Probably change this to

```markdown
    ```shell
    kubectl apply -f https://www.getambassador.io/early-access/yaml/aes-crds.yaml && \
    kubectl wait --for condition=established --timeout=60s crd -lproduct=aes && \
    kubectl apply -f https://www.getambassador.io/early-access/yaml/aes.yaml && \
    kubectl -n ambassador wait --for condition=available --timeout=60s deploy -lproduct=aes
    ```
```

(same as `/user-guide/install`)

Or just link to `/user-guide/install`?